05-online-mentoring.md

```diff
@@ -0,0 +1,25 @@
+# Online Mentoring
+
+---
+
+## Tips
+* Prepare like you would for in-person mentoring. Make sure to go over the content
+the mentee is going to be working on.
+* Introduce yourself! Send a message on the slack channel or do a quick video
+meeting introducing yourself to your mentees. This is a great way to become
+familiar and comfortable around each other.
+* Send check in messages. It is difficult to see if someone is struggling
+with their work while online mentoring, so it's a good idea to check in and
+make sure everything is going smoothly.
+* Encourage screen share and video meetings. If a mentee needs help try to facilitate
+discussion through slack video meetings and screen share. Discourage the use of
+workspace sharing extensions such as CodeTogether and LiveShare, so the mentee can
+navigate the screen themselves.
+* Be accessible. Try to be available to respond quickly to any questions that arise.
+It is perfectly fine to step away from your computer and take a break,
+but make sure to inform your mentees if you are stepping away for a while. If you need
+to step away from your device for an extended period of time make sure to have another
+mentor cover for you for the time being!
+* Schedule sync up times. Allow sufficient time for mentees to work independently,
+and schedule a time for a video meeting where all mentees can discuss their status
+and work through problems together.
```

Comment:
Trying to think if this might incentivize someone to go do other things and just schedule a couple of sync up times. My thought is that being on a video call, even with the video off, would be the most collaborative and make mentors the most available/present. What do you think?

Maybe we can change this tip to say something like this:

`Allow and encourage sufficient time for mentees to work independently. Be available but also take breaks, turn off your video, or work through the material yourself when it looks like mentees have picked up some momentum. No need to hover!`

Needs some refinement, so I welcome your suggestions here.

Reply:
Yeah I think that is good to mention that!

Reply:
I'm not sure if that makes the two tips contradictory, let me know if you think it does.